designer/server/src/test/scala/pl/touk/nussknacker/ui/api/MigrationApiHttpBusinessSpec.scala

```diff
@@ -0,0 +1,160 @@
+package pl.touk.nussknacker.ui.api
+
+import io.circe.syntax._
+import io.restassured.RestAssured.`given`
+import io.restassured.module.scala.RestAssuredSupport.AddThenToResponse
+import org.scalatest.freespec.AnyFreeSpecLike
+import pl.touk.nussknacker.engine.api.graph.ScenarioGraph
+import pl.touk.nussknacker.engine.api.process.ProcessName
+import pl.touk.nussknacker.engine.build.ScenarioBuilder
+import pl.touk.nussknacker.restmodel.validation.ValidationResults.{NodeValidationError, NodeValidationErrorType, ValidationResult}
+import pl.touk.nussknacker.test.base.it.{NuItTest, WithRichConfigScenarioHelper}
+import pl.touk.nussknacker.test.config.WithRichDesignerConfig.TestCategory.Category1
+import pl.touk.nussknacker.test.config.{WithMockableDeploymentManager, WithRichDesignerConfig}
+import pl.touk.nussknacker.test.processes.WithRichScenarioActivitySpecAsserts
+import pl.touk.nussknacker.test.{NuRestAssureExtensions, NuRestAssureMatchers, RestAssuredVerboseLogging}
+import pl.touk.nussknacker.ui.process.marshall.CanonicalProcessConverter
+
+class MigrationApiHttpBusinessSpec
+    extends AnyFreeSpecLike
+    with NuItTest
+    with WithRichDesignerConfig
+    with WithRichConfigScenarioHelper
+    with WithMockableDeploymentManager
+    with WithRichScenarioActivitySpecAsserts
+    with NuRestAssureExtensions
+    with NuRestAssureMatchers
+    with RestAssuredVerboseLogging {
+
+  "The endpoint for scenario migration between environments should" - {
+    "migrate scenario and add update comment" in {
+      given()
+        .applicationState(
+          createSavedScenario(exampleScenario, Category1)
```

Comment:
`Suspicious forward reference in class`
let's declare the examples and other private vals declared after the test as lazy. It will solve the warning